internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -87,6 +88,12 @@ func verifyRefs(what, forwarderName string, status loggingv1.ClusterLogForwarder
 			} else {
 				bad.Insert(ref)
 			}
+		case "filters":
+			if allowed.Has(ref) {
+				good.Insert(ref)
+			} else {
+				bad.Insert(ref)
+			}
 		}
```

Comment:
default:?

Reply:
I don't believe we need a default case here. Originally it was an if/else if statement with no final else clause